webapps/ROOT/index.jsp

```diff
@@ -25,11 +25,11 @@ request.setAttribute("tomcatExamplesUrl", "/examples/");
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta charset="UTF-8" />
+        <meta charset="UTF-8">
```

Comment:
Why did you remove the auto-closing ? It doesn't do any harm. I, personally, prefer the style it was before.

Reply:
In HTML5 you don't need the auto-closing.
I got the style from MDN.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta

Reply:
I know, but it does not do any harm either.
This change makes the diff bigger and harder to back-port to the other branches.
I prefer the XHTML style because it is easier to spot errors in the IDE/text editor without copying to W3C validator in the web.